interface/portraits/portraits_nightelf.gfx

```diff
@@ -11,36 +11,23 @@ spriteTypes = {
 		effectFile = "gfx/FX/portrait.lua"
 		layer = { # GFX_TYPE:[d|p]INDEX:COLOR_LINK:DONT_REFRESH_IF_VALID:CULTURE_INDEX
 			"GFX_character_background:p0"
-			"GFX_empty:p3"
-			"GFX_empty:p5"
-			"GFX_empty:p1:h:y"
-			"GFX_empty:p4:h:y"
-			":p5:c6"
+			"GFX_nightelf_male_clothes_behind:p3"
+			"GFX_nightelf_male_headgear_behind:p5"
+			"GFX_nightelf_male_hair_behind:p1:h:y"
+			"GFX_nightelf_male_beard_behind:p4:h:y"
 			
 			"GFX_nightelf_male_base:p2"
-			"GFX_empty:d0"
-			"GFX_empty:p3"
-			"GFX_empty:d1"
-			"GFX_empty:d2"
-			"GFX_empty:p5"
-			"GFX_empty:d7"	
-			"GFX_empty:d3"
-			"GFX_empty:d6"
-			"GFX_empty:d6:e"
+			"GFX_nightelf_male_face:d2"
+			"GFX_nightelf_male_clothes:p3"
 			
-			"GFX_empty:p7:y"
-			"GFX_empty:p8"
-			"GFX_empty:p9"		
-			"GFX_empty:p10"
+			"GFX_nightelf_male_headgear_mid:p5"	
+			"GFX_nightelf_male_eyes:d6"
+			"GFX_nightelf_male_eyes2:d6:e"
 			
-			"GFX_empty:p4:h:y"
-			"GFX_empty:d4:h"
-			"GFX_empty:p13:y:o32x63"
-			"GFX_empty:p1:h:y"
-			":p5:c7"
-			"GFX_empty:p12:y:o40x45"
-			"GFX_empty:p5"
-			"GFX_empty:p3"
+			"GFX_nightelf_male_hair:p1:h:y"
+			"GFX_nightelf_male_beard:p4:h:y"
```

Comment:
The beard should be behind hairs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/82447653-9fde6180-9ab9-11ea-9a5b-2e2a7e2a97de.png)

</details>